config/routes.rb

```diff
@@ -18,6 +18,7 @@
       default: true) do
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Style/StringLiterals
**Severity**: INFO
**File**: config/routes.rb L18

Prefer single-quoted strings when you don&apos;t need string interpolation or special symbols.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*